sphinxext/opengraph/__init__.py

```diff
@@ -0,0 +1,23 @@
+from typing import Any, Dict
+from sphinx.application import Sphinx
+from .event_handler import html_page_context
+
+# todo: rtd option to grab READTHEDOCS_LANGUAGE and READTHEDOCS_VERSION for link
```

Comment:
We can put these TODOs in a separate PR after this is merged.

Reply:
Yea, those are going to be gone before I remove it from being a draft, just waiting for an opinion on the lists and sections to close that part off